platform-api/scripts/seed_kv.sh

```
@@ -0,0 +1,55 @@
+if [[ "${1:-}" == "--production" ]]; then
+    echo ">>> Seeding PRODUCTION KV namespace"
+    ENVIRONMENT="--env production"
+else
+    echo ">>> Seeding PREVIEW KV namespace (local dev)"
+    ENVIRONMENT="--preview"
+fi
+
+# Build args array to avoid word-splitting issues with ENVIRONMENT
+ARGS=(--namespace-id="$KV_NAMESPACE_ID")
+if [[ -n "$ENVIRONMENT" ]]; then
+    ARGS+=($ENVIRONMENT)
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)